syncthing_gtk/notifications.py

```diff
@@ -164,7 +164,11 @@ def cb_syncthing_folder_progress(self, daemon, folder_id, progress):
 		def cb_syncthing_folder_finished(self, daemon, folder_id):
 			if folder_id in self.syncing:
 				self.syncing.remove(folder_id)
-				self.info(_("Synchronization of folder '%s' is completed.") % (folder_id,))
+                                folder_label = self.app.folders[folder_id]["label"]
```

Comment:
I first tried to use `folders[folder_id].get_title()`, but this will yield the path when no label is set, instead of the ID as fallback.